---

layout: page
title: Representative Publications

---

**\[1\] Genome wide association studies of complex diseases**

M. Chen, J. Cho, H. Zhao (2011) Incorporating biological pathways via a
Markov random field model in genome-wide association studies. *PLoS
Genetics*, 7: e1001353.

L. Hou, M. Chen, C. K. Zhang, J. Cho, H. Zhao (2014) Guilt by Rewiring:
Gene prioritization through network rewiring in genome wide association
studies. *Human Molecular Genetics*, 23: 2780-2790.

D. Chung, C. Yang, C. Li, J. Gelernter, H. Zhao (2014) GPA: A
statistical approach to prioritizing GWAS results by integrating
pleiotropy and annotation. *PLOS Genetics*, 10: e1004787.

J. Jiang, C. Li, D. Paul, C. Yang, H. Zhao (2016) On high-dimensional
misspecified mixed model analysis in genome-wide association study.
*Annals of Statistics*, 44: 2127–2160.

Q. Lu, R. Powles, Q. Wang, J. He, H. Zhao (2016) Integrative
tissue-specific functional annotations in the human genome provide novel
insights on many complex traits and improve signal prioritization in
genome wide association studies. *PLOS Genetics*, 12: e1005947.

D. Chung, H. J. Kim, H. Zhao (2017) graph-GPA: A graphical model for
prioritizing GWAS results and investigating pleiotropic architecture.
*PLOS Computational Biology*, 13: e1005388.

Q. Lu, R. L. Powles, S. Abdallah, D. Ou, Q. Wang, Y. Hu, Y. Lu, W. Liu,
B. Li, S. Mukherjee, P. K. Crane, H. Zhao (2017) Systematic
tissue-specific functional annotation of the human genome highlights
immune-related DNA elements for late-onset Alzheimer’s disease. *PLOS
Genetics*, 13: e1006933.

Q. Lu, B. Li, D. Ou, M. Erlendsdottir, R. L. Powles, T. Jiang, Y. Hu, D.
Chang, C. Jin, W. Dai, Q. He, Z. Liu, S. Mukherjee, P. K. Crane, H. Zhao
(2017) A powerful approach to estimating annotation-stratified genetic
covariance using GWAS summary statistics. *American Journal of Human
Genetics*, 101: 939-964.

Y. Hu, M. Li, Q. Lu, H. Weng, J. Wang, S. M. Zekavat, Z. Yu, B. Li, J.
Gu, S. Muchnik, Y. Shi, B. W. Kunkle, S. Mukherjee, P. Natarajan, A.
Naj, A. Kuzma, Y. Zhao, P. K. Crane, Alzheimer's Disease Genetics
Consortium, H. Lu, H. Zhao (2019) A statistical framework for
cross-tissue transcriptome-wide association analysis. *Nature Genetics*,
51: 568-576.

**\[2\] Genetic risk predictions**

C. Li, C. Yang, J. Gelernter, H. Zhao (2014) Improving genetic risk
prediction by leveraging pleiotropy. *Human Genetics*, 133: 639-650.

Y. Hu, Q. Lu, R. Powles, X. Yao, C. Yang, F. Fang, X. Xu, H. Zhao (2017)
Leveraging functional annotations in genetic risk prediction for human
complex diseases. *PLOS Computational Biology*, 13: e1005589.

Y. Hu, Q. Lu, W. Liu, Y. Zhang, M. Li, H. Zhao (2017) Joint modeling of
genetically correlated diseases and functional annotations increases
accuracy of polygenic risk prediction. *PLOS Genetics,* 13: e1006836.

S. Song, W. Jiang, L. Hou, H. Zhao (2020) Leveraging effect size
distributions to improve polygenic risk scores derived from summary
statistics of genome-wide association studies. *PLOS Computational
Biology*, 16: e1007565.

**\[3\] Whole exome and whole genome sequencing analysis**

S. Zaidi, M. Choi, H. Wakimoto, L. Ma, J. Jiang, J. D. Overton, A.
Romano-Adesman, R. D. Bjornson, R. E. Breitbart, K. K. Brown, N. J.
Carriero, Y. H. Cheung, J. Deanfield, S. DePalma, K. A. Fakhro, J.
Glessner, H. Hakonarson, M. J. Italia, J. R. Kaltman, J. Kaski, R. Kim,
J. K. Kline, T. Lee, J. Leipzig, A. Lopez, S. M. Mane, L. E. Mitchell,
J. W. Newburger, M. Parfenov, I. Pe'er, G. Porter, A. E. Roberts, R.
Sachidanandam, S. J. Sanders, H. S. Seiden, M. W. State, S. Subramanian,
I. R. Tikhonova, W. Wang, D. Warburton, P. S. White, I. A. Williams, H.
Zhao, J. G. Seidman, M. Brueckner, W. K. Chung, B. D. Gelb, E.
Goldmuntz, C. E. Seidman, R. P. Lifton (2013) De novo mutations in
histone-modifying genes in congenital heart disease. *Nature*, 498:
220-223.

S. C. Jin, J. Homsy, S. Zaidi, Q. Lu, S. Morton, S. R. DePalma, X. Zeng,
H. Qi, W. Chang, W.-C. Hung, M. C. Sierant, S. Haider, J. Zhang, J.
Knight, R. D. Bjornson, C. Castaldi, I. R. Tikhonoa, K. Bilguvar, S. M.
Mane, S. J. Sanders, S. Mital, M. Russell, W. Gaynor, J. Deanfield, A.
Giardini, G. A. Porter Jr., D. Srivastava, C. W. Lo, Y. Shen, W. S.
Watkins, M. Yandell, H. J. Yost, M. Tristani-Firouzi, J. W. Newburger,
A. E. Roberts, R. Kim, H. Zhao, J. R. Kaltman, E. Goldmuntz, W. K.
Chung, J. G. Seidman, B. D. Gelb, C. E. Seidman2, R. P. Lifton, M.
Brueckner (2017)

Contribution of rare inherited and de novo variants among 2,871
congenital heart disease probands. *Nature Genetics*, 49: 1593-1601.

**\[4\] Cancer genomics**

Y. Zhang, Z. Ouyang, H. Zhao (2017) A statistical framework for data
integration through graphical models with application to cancer
genomics. *Annals of Applied Statistics,* 11: 161-184.

L. Zeng, J. L. Warren, H. Zhao (2019) Phylogeny-based tumor subclone
identification using a Bayesian feature allocation model. *Annals of
Applied Statistics*, 13: 1212–1241.

D. Tang, S. Park, H. Zhao (2019) NITUMID: Nonnegative Matrix
Factorization-based Immune-TUmor MIcroenvironment Deconvolution.
*Bioinformatics*, 36: 1344-1350.

S. Park, H. Xu, H. Zhao (2020) Integrating multidimensional data for
clustering analysis with applications to cancer patient data. *Journal
of American Statistical Association*, in press.

**\[5\] Neuroscience**

Z. Lin, S. Sanders, M. Li, N. Sestan, M. State, H. Zhao (2015) A Markov
random field-based approach to characterizing human brain development
using spatial-temporal transcriptome data. *Annals of Applied
Statistics*, 9: 429–451.

Y. Zhu, A. M. M. Sousa, T. Gao, M. Skarica, M. Li, G. Santpere, P.
Esteller-Cucala, D. Juan, L. Ferrández-Peral, F. O. Gulden, M. Yang, D.
J. Miller, T. Marques-Bonet, Y. Imamura Kawasawa, H. Zhao, N. Sestan
(2018) Spatiotemporal transcriptomic divergence across human and macaque
brain development. *Science* 362(6420).

**\[6\] Single cell analysis**

S. Park, H. Zhao (2018) Spectral clustering based on learning similarity
matrix. *Bioinformatics*, 34: 2069-2076.

Y. Liu, J. Warren, H. Zhao (2019) A hierarchical Bayesian model for
single-cell clustering using RNA-sequencing data. *Annals of Applied
Statistics*, Volume 13: 1733-1752.

**\[7\] Microbiome analysis**

T. Wang, H. Zhao (2017) Structured subcomposition selection in
regression and its application to microbiome data analysis. *Annals of Applied Statistics,* 11: 771-791.

T. Wang, H. Zhao (2017) A Dirichlet-tree multinomial regression model
for associating dietary nutrients with gut microorganisms. *Biometrics*,
73: 792-801.

T. Wang, H. Zhao (2017) Constructing predictive microbial signatures at
multiple taxonomic levels. *Journal of the American Statistical
Association*, 112: 1022-1031.

T. Wang, C. Yang, H. Zhao (2019) Prediction analysis for microbiome
sequencing data. *Biometrics*, 75: 875-884.

**\[8\] Network models**

B. Li, H. Chun, H. Zhao (2012) Sparse estimation of conditional
graphical models with application to gene networks. *Journal of American
Statistical Association*, 107: 152-167.

B. Li, H. Chun, H. Zhao (2014) On an additive semi-graphoid model for
statistical networks with application to pathway analysis. *Journal of
American Statistical Association*, 109: 1188-1204.

H. Chun, X. Zhang, H. Zhao (2015) Gene regulation network inference with
joint sparse Gaussian graphical models. *Journal of Computational and
Graphical Statistics*, 24: 954–974*.*

Y. Hu, H. Zhao (2016) CCor: a whole genome network-based similarity
measure between two genes. *Biometrics*, 72: 1216-1225.

Z. Lin, T. Wang, C. Yang, H. Zhao (2017) On joint estimation of Gaussian
graphical models for spatial and temporal data. *Biometrics*, 73:
769-779.

K. Lee, B. Li, H. Zhao (2016) On an additive partial correlation
operator and nonparametric estimation of graphical models. *Biometrika*,
103: 513-530.

K-Y Lee, T. Liu, B. Li, H. Zhao (2020) Learning causal networks via
additive faithfulness. *Journal of Machine Learning Research*, in press.
