---

layout: page
title: Softwares

---

<!DOCTYPE html>
<!-- saved from url=(0120)#Transcription%20Factor%20Binding%20using%20Phylogeny -->
<html><head><link rel="stylesheet" href="{{ site.baseurl }}{% link assets/css/softwares.css %}"> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">  </head><body data-gr-c-s-loaded="true">


<ul class="tab1">
  <li1><a target="_blank" class="tablinks active" onclick="openblock(event, 0)" style="text-align:center">Main Softwares</a></li1>
  <li1><a target="_blank" class="tablinks" onclick="openblock(event, 1)" style="text-align:center">Other Softwares</a></li1>
</ul>

<div id="other" class="tabcontent" display=block>
<a href="#Human%20Genome%20Annotations">Human Genome Annotations<br></a>
<a href="#Analysis%20of%20Genome%20Wide%20Association%20Study%20Data">Analysis of Genome Wide Association Study Data<br></a>
<a href="#eQTL">eQTL<br></a>
<a href="#Pathway%20Analysis">Pathway Analysis<br></a>
<a href="#Cancer%20Mutation%20Cluster%20Identifications">Cancer Mutation Cluster Identifications<br></a>
<h3 id="Human Genome Annotations">Human Genome Annotations</h3>
<ul>
	<li>GenoCanyon&nbsp;(<a target="_blank" href="http://genocanyon.med.yale.edu/index.html">Server</a>) (<a target="_blank" href="http://genocanyon.med.yale.edu:7772/">Web Application</a>)
A statistical framework to predict functional non-coding regions in the human genome through integrated analysis of genomic conservation measures and biochemical annotations.</li>
</ul>
<ul>
	<li>GenoSkyline&nbsp;(<a target="_blank" href="http://genocanyon.med.yale.edu/GenoSkyline">Website</a>)
Tissue-specific functional annotation through integrative analysis of Roadmap epigenomic data</li>
</ul>
<ul>
	<li>GenoSkyline&nbsp;Plus(<a target="_blank" href="http://genocanyon.med.yale.edu/GenoSkyline">Website</a>)
GenoSkyline-Plus is a comprehensive update of GenoSkyline that incorporates more annotation data into the framework and extends to 127 integrated annotation tracks covering a spectrum of human tissue and cell types. </li>
</ul>
<h3 id="Analysis of Genome Wide Association Study Data">Analysis of Genome Wide Association Study Data</h3>
<ul>
	<li>GWAS.PC (<a target="_blank" href="http://genocanyon.med.yale.edu/Software_Zhaocenter/GWASPC/example.html">User Manual</a> <a target="_blank" href="http://genocanyon.med.yale.edu/Software_Zhaocenter/GWASPC/GWAS.PC_1.0.tar.gz">Code</a>)
GWAS.PC (GWAS Power Calculation) is an R package that does power analysis in genome wide association studies. In particular, genotyping error is considered in power calculation. </li>
</ul>
<ul>
	<li>GenoWAP&nbsp;(<a target="_blank" href="http://genocanyon.med.yale.edu/GenoWAP">Website</a>)
Post-GWAS prioritization through integrated analysis of GWAS summary statistics and GenoCanyon genomic functional annotation</li>
</ul>
<ul>
	<li>GPA&nbsp;(<a target="_blank" href="http://dongjunchung.github.io/GPA/">Website</a>)
A statistical approach to prioritizing GWAS results by integrating pleiotropy information and annotation data.</li>
</ul>
<ul>
	<li>GBR (<a target="_blank" href="http://zhaocenter.org/labcode/20130705_GBR/gbr_1.0.tar.gz" download="">Download</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130705_GBR/example.zip" download="">Example</a>)
This is an R package implementing a post-GWAS prioritization algorithm, which incorporates the rewiring information of co-expression network to prioritize GWAS signals.</li>
</ul>
<ul>
	<li>GWAS with MRF pathway&nbsp; (<a target="_blank" href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1001353">Related Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/20110611PathwayMRF/example.R" download="">Example.R</a> <a target="_blank" href="http://zhaocenter.org/labcode/20110611PathwayMRF/fun_network.R" download="">fun_network.R</a> <a target="_blank" href="http://zhaocenter.org/labcode/20110611PathwayMRF/network.csv" download="">network.csv</a> <a target="_blank" href="http://zhaocenter.org/labcode/20110611PathwayMRF/pval.txt" download="">pval.txt</a>)
This is a program to implement the Markov Random Field (MRF) method to incorporate pathway topology for genome wide association studies.</li>
</ul>
<h3 id="eQTL">eQTL</h3>
<ul>
	<li>LORS (<a target="_blank" href="http://zhaocenter.org/labcode/20120501_LORS/LORS.zip" download="">Download</a>)
A Low-Rank representation and Sparse regression for eQTL mapping. This algorithm accounts for confounding factors such as unobserved covariates, experimental artifacts, and unknown environmental perturbations.</li>
</ul>
<h3 id="Pathway Analysis">Pathway Analysis</h3>
<ul>
	<li>Pathway Analysis&nbsp;using Random Forests (<a target="_blank" href="http://people.duke.edu/~hp44/r_code.htm">Source code and supplemental data</a>)
An R code for pathway-based classification and regression using Random Forests.</li>
</ul>
<ul>
	<li>GRAPE&nbsp;(<a target="_blank" href="http://genocanyon.med.yale.edu/Software_Zhaocenter/GRAPE.zip">Source code and sample datasets</a>)
GRAPE is a template method that allows for identification of perturbed pathways in individual tumor samples relative to a reference collection of samples (e.g., matched healthy tissue). GRAPE is sensitive to biological variability, robust to batch effects and can be applied to any gene expression platform.</li>
</ul>
<ul>
	<li>COSINE&nbsp; (<a target="_blank" href="http://bioinformatics.oxfordjournals.org/content/27/9/1290">Related Paper</a>) (<a target="_blank" href="http://cran.r-project.org/web/packages/COSINE/index.html">Software</a>)
<div>An R package to extract the globally most discriminative sub-network from multiple gene expression data sets with integration of protein-protein interactions data.</div></li>
</ul>
<h3 id="Cancer Mutation Cluster Identifications">Cancer Mutation Cluster Identifications</h3>
<ul>
	<li>iPAC (<a target="_blank" href="http://www.bioconductor.org/packages/devel/bioc/html/iPAC.html">More information</a>)
iPAC (Identification of Protein Amino acid mutation Clustering) finds mutation clusters on the amino acid level while taking into account the protein structure.</li>
</ul>
<ul>
	<li>GraphPAC (<a target="_blank" href="http://www.bioconductor.org/packages/2.12/bioc/html/GraphPAC.html">More information</a>)
A bioconductor R package for identifying mutational clusters of amino acids in a protein while utilizing the protein tertiary structure via a graph theoretical model.</li>
</ul>
</div>

<div id="other" class="tabcontent" style="display: none;">
<a href="#Pathway%20Tools">Pathway Tools<br></a>
<a href="#Drug%20Target%20Identifications">Drug Target Identifications<br></a>
<a href="#Haplotype%20Analysis">Haplotype Analysis<br></a>
<a href="#TDT">TDT<br></a>
<a href="#Linkage%20Disequilibrium%20Mapping">
Linkage Disequilibrium Mapping<br></a>
<a href="#Data%20Integration">
Data Integration<br></a>
<a href="#Pathway%20Tools">Pathway Tools<br></a>
<a href="#Population%20Structure">Population Structure<br></a>
<a href="#Cluster%20analysis">Cluster analysis<br></a>
<a href="#Sequencing%20Data%20Analysis">Sequencing Data Analysis<br></a>
<a href="#Genotype%20Calling%20from%20Microarrays">Genotype Calling from Microarrays<br></a>
<a href="#MicroRNA-gene%20interactions">MicroRNA-gene interactions<br></a>
<a href="#Transcription%20Factor%20Binding%20using%20Phylogeny">Transcription Factor Binding using Phylogeny<br></a>
<a href="http://zhaocenter.org/wp-content/uploads/2017/12/Proteomics%20Data%20Analysis">Proteomics Data Analysis<br></a>
<a href="http://zhaocenter.org/wp-content/uploads/2017/12/Microbiome%20Analysis">Microbiome Analysis<br></a>
<a href="#Human%20Nondisjunciton%20Data">Human Nondisjunciton Data<br></a>
<a href="#Modifier%20Gene%20Analysis">Modifier Gene Analysis<br></a>
<a href="#Others">Others<br></a>
<h3 id="Pathway Tools">Pathway Tools</h3>
<ul>
	<li>VitaPad&nbsp; (<a target="_blank" href="http://vitapad.sourceforge.net/">Link to the software</a>)
A program to visualize and analyze pathways.</li>
</ul>
<h3 id="Drug Target Identifications">Drug Target Identifications</h3>
<ul>
	<li>iFAD (<a target="_blank" href="http://cran.r-project.org/web/packages/iFad/index.html">Software</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/20120404_iFad/NCI-60%20data.7z">DataSet</a>)
<div>An R package implementing a bayesian sparse factor model for the joint analysis of paired datasets, the gene expression and drug sensitivity profiles, measured across the same panel of samples, e.g. cell lines.</div></li>
</ul>
<ul>
	<li>FacPad (<a target="_blank" href="http://cran.r-project.org/web/packages/FacPad/index.html">Software</a>) (<a target="_blank" href="http://zhaocenter.org/facpad/">More Information</a>)
<div>&nbsp;An R package implementing a Bayesian sparse factor model for the inference of pathways responsive to drug treatment.</div></li>
</ul>
<h3 id="Haplotype Analysis">Haplotype Analysis</h3>
<ul>
	<li>Haplore (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/15231536">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/Haplore/haplore_help.pdf" download="">ReadMe - PDF</a>)&nbsp; (<a target="_blank" href="http://zhaocenter.org/labcode/Haplore/haplore_v242_win.exe" download="">Download-Windows</a> <a target="_blank" href="http://zhaocenter.org/labcode/Haplore/haplore_v242_unix.gz" download="">Download-Solaris</a> <a target="_blank" href="http://zhaocenter.org/labcode/Haplore/haplore_v242_linux.gz" download="">Download-Linux</a>)
A Program for Haplotype Reconstruction in Pedigrees</li>
</ul>
<ul>
	<li>Overall LD (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/10738528">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/OverallLDEstimator/overall.zip">Download</a>)
A program to assess the overall Linkage Disequilibrium (LD) over several markers and assess the LD between groups of markers.</li>
</ul>
<h3 id="TDT">TDT</h3>
<ul>
	<li>Multiple TDT (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/10968775">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/multitdt.zip" download="">Download</a>)
A program to do the associate test (TDT type test for nuclear-family data) using several tightly linked markers simultaneously</li>
</ul>
<ul>
	<li>QSAT (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/11479834">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/QSAT/QSATrm.pdf" download="">ReadMe - PDF</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/QSAT/QSAT.zip" download="">Download</a>)
A Program for Quantitative Association Analysis</li>
</ul>
<h3 id="Linkage Disequilibrium Mapping">Linkage Disequilibrium Mapping</h3>
<ul>
	<li>LD Mapping with Haplotype Data (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/11055377">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/LDMhap.zip" download="">Download</a>)
A program of mapping disease genes (including location and confidence interval) by using haplotypes with disease mutation and the haplotypes without disease mutation (or allele frequencies in normal population).</li>
</ul>
<ul>
	<li>LD Mapping with Genotype Data (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/11754474">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/ldmgen.zip" download="">Download</a>)
This is a program to mapping the disease gene (including location and confidence interval) by using multisite genotype data of case-control studies.</li>
</ul>
<h3 id="Data Integration">Data Integration</h3>
<ul>
	<li>MDIntegration (<a target="_blank" href="http://web.hku.hk/~herbpang/MDIntegration.html">More information</a>) 
An R package for multi data sources integration using composite association network, relevance vector machine (RVM), or RVM-boost.</li>
</ul>
<h3 id="Population Structure">Population Structure</h3>
<ul>
	<li>Population Structure Inference (PSMix) (<a target="_blank" href="http://zhaocenter.org/labcode/PSMix/psmixreadme.txt" download="">Readme</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/PSMix/PSMix_1.1-2.zip">Download</a>)
An R package to infer population structure using genomic markers.</li>
</ul>
<h3 id="Cluster analysis">Cluster analysis</h3>
<ul>
	<li>BCLUST (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/11793234">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/Bclust/bclust_help.pdf" download="">ReadMe - PDF</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/Bclust/bclust.zip" download="">Download</a>)
A program to assess reliability of gene clusters from expression data by using consensus tree and bootstrap resampling method</li>
</ul>
<h3 id="Sequencing Data Analysis">Sequencing Data Analysis</h3>
<ul>
	<li>RNASeqBias (<a target="_blank" href="http://zhaocenter.org/labcode/20110309RNAseqRPackage/RNASeqBias_1.0.tar.gz" download="">Download</a> <a target="_blank" href="http://zhaocenter.org/labcode/20110309RNAseqRPackage/RNASeqBias-manual.pdf" download="">Manual</a> <a target="_blank" href="http://zhaocenter.org/labcode/20110309RNAseqRPackage/vignette.pdf" download="">Vignette</a>)
An R package for detecting and correcting biases in RNA-Sequencing data. The download link contains the package, the manual and a "vignette" for quick start.</li>
</ul>
<ul>
	<li>pairedBayes (<a target="_blank" href="http://zhaocenter.org/labcode/20130514_pairedBayes/pairedBayesRsource.R" download="">Download</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130514_pairedBayes/ReadMe.txt" download="">ReadMe.txt</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130514_pairedBayes/pairedBayesExample2.Rdat" download="">Example Rdat file</a>)
An R code for Bayesian modeling of paired RNA-seq experiment.</li>
</ul>
<ul>
	<li>Psafe (<a target="_blank" href="http://zhaocenter.org/labcode/20120530_Psafe/Psafe.zip" download="">Download</a>)
A pipeline to adjust the biases and errors of sequencing data to gain more accurate allele frequency estimation.</li>
</ul>
<h3 id="Genotype Calling from Microarrays">Genotype Calling from Microarrays</h3>
<ul>
	<li>M3 and M3-S (<a target="_blank" href="http://genocanyon.med.yale.edu/Software_Zhaocenter/M3_and_M3_S.tar.gz" download="">Download</a>)
A combined package for M3 and M3-S - an improved SNP calling algorithm for Illumina BeadArray Data.</li>
</ul>
<h3 id="MicroRNA-gene interactions">MicroRNA-gene interactions</h3>
<ul>
	<li>MCMG (<a target="_blank" href="http://zhaocenter.org/labcode/20130701_MCMG/MCMG.zip" download="">Download</a>)
MCMG is designed to infer the gene-microRNA interaction pairs from joint analysis of multiple cancers.</li>
</ul>
<h3 id="Transcription Factor Binding using Phylogeny">Transcription Factor Binding using Phylogeny</h3>
<ul>
	<li>TFBphylo (<a target="_blank" href="http://zhaocenter.org/labcode/20120316TFBphylo/TFBphylo_1.0.tar.gz" download="">Download</a>)
An R package implementing a phylogenetic tree based method to model the on/off rates of transcription factor binding (TFB) events, and uses the EM algorithm to estimate model parameters.</li>
</ul>
<h3 id="Proteomics Data Analysis">Proteomics Data Analysis</h3>
<ul>
	<li>MRM pre-processing (<a target="_blank" href="http://zhaocenter.org/labcode/20131227_MRM/Rscript_for_Data_Preprocessing_MRM_Chung_Colangelo_Zhao.R" download="">Download</a>)
Workflow for data pre-processing of multiple reaction monitoring (MRM) experiments. It includes data quality assessment and normalization.</li>
</ul>
<ul>
	<li>Graphics for MRM and SWATH (<a target="_blank" href="http://zhaocenter.org/labcode/20130221GraphicsMRMandSWATH/MRMandSWATHgraphics_1.0.tar.gz" download="">Download</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130221GraphicsMRMandSWATH/MRMandSWATHgraphics.pdf" download="">MRM &amp; SWATH Graphics Manual</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130221GraphicsMRMandSWATH/iTRAQ_Normalization_FoldChange.R" download="">iTRAQ code</a> <a target="_blank" href="http://zhaocenter.org/labcode/20130221GraphicsMRMandSWATH/Manual_iTRAQ_Normalization_FoldChange.pdf" download="">iTRAQ Manual</a>)
An R package providing exploratory graphics and qualify assessment for MRM data generated from Multiquant (ABSciex) and SWATH datasets.</li>
</ul>
<ul>
	<li>MSq (<a target="_blank" href="http://zhaocenter.org/labcode/MSQ/MSq_1.0.tar.gz">Download</a>)
Data Pre-processing for label-free multiple reaction monitoring (MRM) experiments (Quality assessment and data normalization)</li>
</ul>
<h3 id="Microbiome Analysis">Microbiome Analysis</h3>
<ul>
	<li>MSClust (<a target="_blank" href="http://zhaocenter.org/labcode/20130708_MSClust/MSClust.rar" download="">Download</a>)
An M package for Clustering 16S rRNA sequences into OTUs. The download link contain the package and some benchmark data sets.</li>
</ul>
<h3 id="Human Nondisjunciton Data">Human Nondisjunciton Data</h3>
<ul>
	<li>EM based method (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=li+j*+zhao+h*+trisomy">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/em.zip">Download</a>)
An EM based algorithm to estimate genetic distances and chiasma distribution for human UPD and trisomy data.</li>
</ul>
<ul>
	<li>HMM based method (<a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/11764246">Paper</a>) (<a target="_blank" href="http://zhaocenter.org/labcode/chi.zip">Download</a>)
A Hidden Markov model based algorithm to estimate genetic distances and chiasma distribution for human UPD and trisomy data.</li>
</ul>
<h3 id="Modifier Gene Analysis">Modifier Gene Analysis</h3>
<ul>
	<li>siliColony (<a target="_blank" href="http://zhaocenter.org/labcode/siliColony.zip">Download</a>)
A program to estimate the probability that a modifier gene is responsible for the segregation of phenotypes.</li>
</ul>
<h3 id="Others">Others</h3>
<ul>
	<li>viRF(<a target="_blank" href="http://zhaocenter.org/wp-content/uploads/2017/03/viRF_code.zip">Download</a>)
Variable Importance-weighted Random Forests.</li>
</ul>
</div>
     
<div id="UMS_TOOLTIP" style="position: absolute; cursor: pointer; z-index: 2147483647; background: transparent; top: -100000px; left: -100000px;"></div><umsdataelement id="UMSSendDataEventElement"></umsdataelement></body></html> 
