<!doctype html>
<!--[if lt IE 7]><html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if (IE 7)&!(IEMobile)]><html class="no-js lt-ie9 lt-ie8" lang="en"><![endif]-->
<!--[if (IE 8)&!(IEMobile)]><html class="no-js lt-ie9" lang="en"><![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"><!--<![endif]-->
<head>
<meta charset="utf-8">


<title>Representative Publications &#8211; Zhao Lab</title>

<meta name="keywords" content="">


<!-- Twitter Cards -->
<meta name="twitter:title" content="Representative Publications">

<meta name="twitter:site" content="@hongyuzhao2">
<meta name="twitter:creator" content="@hongyuzhao2">

<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="/images/default-thumb.png">

<!-- Open Graph -->
<meta property="og:locale" content="en_US">
<meta property="og:type" content="article">
<meta property="og:title" content="Representative Publications">

<meta property="og:url" content="/research/">
<meta property="og:site_name" content="Zhao Lab">

<meta property="og:image" content="/images/default-thumb.png">






<link rel="canonical" href="/research/">
<link href="/feed.xml" type="application/atom+xml" rel="alternate" title="Zhao Lab Feed">

<!-- https://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<!-- For all browsers -->
<link rel="stylesheet" href="/assets/css/main.css">


<meta http-equiv="cleartype" content="on">


<!-- Modernizr -->
<script src="/assets/js/vendor/modernizr-2.7.1.custom.min.js"></script>


<link href='//fonts.googleapis.com/css?family=PT+Sans+Narrow:400,700%7CPT+Serif:400,700,400italic' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="/assets/css/academicons/css/academicons.css"/>
<!-- Icons -->
<!-- 16x16 -->
<link rel="shortcut icon" href="/favicon.ico">
<!-- 32x32 -->
<link rel="shortcut icon" href="/favicon.png">
<!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
<link rel="apple-touch-icon-precomposed" href="/images/apple-touch-icon-precomposed.png">
<!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
<link rel="apple-touch-icon-precomposed" sizes="72x72" href="/images/apple-touch-icon-72x72-precomposed.png">
<!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
<link rel="apple-touch-icon-precomposed" sizes="114x114" href="/images/apple-touch-icon-114x114-precomposed.png">
<!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="/images/apple-touch-icon-144x144-precomposed.png">

<!--Jekyll-seo plugin-->
<!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Representative Publications | Zhao Lab</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Representative Publications" />
<meta property="og:locale" content="en_US" />
<link rel="canonical" href="http://localhost:4000/research/" />
<meta property="og:url" content="http://localhost:4000/research/" />
<meta property="og:site_name" content="Zhao Lab" />
<script type="application/ld+json">
{"@type":"WebPage","url":"http://localhost:4000/research/","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/yale-blue.png#%20filename%20of%20a%20logo%20image%20that%20has%20been%20placed%20into%20your%20images%20directory"}},"headline":"Representative Publications","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


</head>

<body class="page">

<!--[if lt IE 9]><div class="browser-upgrade alert alert-info">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div><![endif]-->

<div class="navigation-wrapper">
	<div class="site-name">
		
		<a href="/"><img src="/images/yale-blue.png# filename of a logo image that has been placed into your images directory ">&nbsp;&nbsp;&nbsp;Zhao Lab</a>
		
	</div><!-- /.site-name -->
	<div class="top-navigation">
		<nav role="navigation" id="site-nav" class="nav">
		    <ul>
		        
				    
				    <li><a href="/" >Home</a></li>
				
				    
				    <li><a href="/research/" >Research</a></li>
				
				    
				    <li><a href="/members/" >Members</a></li>
				
				    
				    <li><a href="/publications/" >Publications</a></li>
				
				    
				    <li><a href="/softwares/" >Software</a></li>
				
		    </ul>
		</nav>
	</div><!-- /.top-navigation -->
</div><!-- /.navigation-wrapper -->




<div id="main" role="main">
  <div class="article-author-side">
    

<div itemscope itemtype="https://schema.org/Person">


	<img src="/images/bio-photo.jpg" class="bio-photo" alt="Hongyu Zhao, Ph.D. bio photo">


  <h3 itemprop="name">Hongyu Zhao, Ph.D.</h3>
  <p>Department Chair and Ira V. Hiscock Professor of Biostatistics, Professor of Genetics and Professor of Statistics and Data Science</p>
  <a href="mailto:hongyu.zhao@yale.edu" class="author-social" target="_blank"><i class="fa fa-fw fa-envelope-square"></i> Email</a>
  <a href="https://twitter.com/hongyuzhao2" class="author-social" target="_blank"><i class="fa fa-fw fa-twitter-square"></i> Twitter</a>
  
  
  
  
  
  
  
  <a href="https://github.com/hongyuzhao68" class="author-social" target="_blank"><i class="fa fa-fw fa-github"></i> Github</a>
  
  
  
  
  
  
  
  
  
  
  
  
  
  <a href="https://orcid.org/0000-0003-1195-9607" class="author-social" target="_blank"><i class="ai ai-fw ai-orcid"></i> ORCID</a>
</div>

  </div>
  <article class="page">
    <h1>Representative Publications</h1>
    <div class="article-wrap">
      <p><strong>[1] Genome wide association studies of complex diseases</strong></p>

<p>M. Chen, J. Cho, H. Zhao (2011) Incorporating biological pathways via a
Markov random field model in genome-wide association studies. <em>PLoS
Genetics</em>, 7: e1001353.</p>

<p>L. Hou, M. Chen, C. K. Zhang, J. Cho, H. Zhao (2014) Guilt by Rewiring:
Gene prioritization through network rewiring in genome wide association
studies. <em>Human Molecular Genetics</em>, 23: 2780-2790.</p>

<p>D. Chung, C. Yang, C. Li, J. Gelernter, H. Zhao (2014) GPA: A
statistical approach to prioritizing GWAS results by integrating
pleiotropy and annotation. <em>PLOS Genetics</em>, 10: e1004787.</p>

<p>J. Jiang, C. Li, D. Paul, C. Yang, H. Zhao (2016) On high-dimensional
misspecified mixed model analysis in genome-wide association study.
<em>Annals of Statistics</em>, 44: 2127–2160.</p>

<p>Q. Lu, R. Powles, Q. Wang, J. He, H. Zhao (2016) Integrative
tissue-specific functional annotations in the human genome provide novel
insights on many complex traits and improve signal prioritization in
genome wide association studies. <em>PLOS Genetics</em>, 12: e1005947.</p>

<p>D. Chung, H. J. Kim, H. Zhao (2017) graph-GPA: A graphical model for
prioritizing GWAS results and investigating pleiotropic architecture.
<em>PLOS Computational Biology</em>, 13: e1005388.</p>

<p>Q. Lu, R. L. Powles, S. Abdallah, D. Ou, Q. Wang, Y. Hu, Y. Lu, W. Liu,
B. Li, S. Mukherjee, P. K. Crane, H. Zhao (2017) Systematic
tissue-specific functional annotation of the human genome highlights
immune-related DNA elements for late-onset Alzheimer’s disease. <em>PLOS
Genetics</em>, 13: e1006933.</p>

<p>Q. Lu, B. Li, D. Ou, M. Erlendsdottir, R. L. Powles, T. Jiang, Y. Hu, D.
Chang, C. Jin, W. Dai, Q. He, Z. Liu, S. Mukherjee, P. K. Crane, H. Zhao
(2017) A powerful approach to estimating annotation-stratified genetic
covariance using GWAS summary statistics. <em>American Journal of Human
Genetics</em>, 101: 939-964.</p>

<p>Y. Hu, M. Li, Q. Lu, H. Weng, J. Wang, S. M. Zekavat, Z. Yu, B. Li, J.
Gu, S. Muchnik, Y. Shi, B. W. Kunkle, S. Mukherjee, P. Natarajan, A.
Naj, A. Kuzma, Y. Zhao, P. K. Crane, Alzheimer’s Disease Genetics
Consortium, H. Lu, H. Zhao (2019) A statistical framework for
cross-tissue transcriptome-wide association analysis. <em>Nature Genetics</em>,
51: 568-576.</p>

<p><strong>[2] Genetic risk predictions</strong></p>

<p>C. Li, C. Yang, J. Gelernter, H. Zhao (2014) Improving genetic risk
prediction by leveraging pleiotropy. <em>Human Genetics</em>, 133: 639-650.</p>

<p>Y. Hu, Q. Lu, R. Powles, X. Yao, C. Yang, F. Fang, X. Xu, H. Zhao (2017)
Leveraging functional annotations in genetic risk prediction for human
complex diseases. <em>PLOS Computational Biology</em>, 13: e1005589.</p>

<p>Y. Hu, Q. Lu, W. Liu, Y. Zhang, M. Li, H. Zhao (2017) Joint modeling of
genetically correlated diseases and functional annotations increases
accuracy of polygenic risk prediction. <em>PLOS Genetics,</em> 13: e1006836.</p>

<p>S. Song, W. Jiang, L. Hou, H. Zhao (2020) Leveraging effect size
distributions to improve polygenic risk scores derived from summary
statistics of genome-wide association studies. <em>PLOS Computational
Biology</em>, 16: e1007565.</p>

<p><strong>[3] Whole exome and whole genome sequencing analysis</strong></p>

<p>S. Zaidi, M. Choi, H. Wakimoto, L. Ma, J. Jiang, J. D. Overton, A.
Romano-Adesman, R. D. Bjornson, R. E. Breitbart, K. K. Brown, N. J.
Carriero, Y. H. Cheung, J. Deanfield, S. DePalma, K. A. Fakhro, J.
Glessner, H. Hakonarson, M. J. Italia, J. R. Kaltman, J. Kaski, R. Kim,
J. K. Kline, T. Lee, J. Leipzig, A. Lopez, S. M. Mane, L. E. Mitchell,
J. W. Newburger, M. Parfenov, I. Pe’er, G. Porter, A. E. Roberts, R.
Sachidanandam, S. J. Sanders, H. S. Seiden, M. W. State, S. Subramanian,
I. R. Tikhonova, W. Wang, D. Warburton, P. S. White, I. A. Williams, H.
Zhao, J. G. Seidman, M. Brueckner, W. K. Chung, B. D. Gelb, E.
Goldmuntz, C. E. Seidman, R. P. Lifton (2013) De novo mutations in
histone-modifying genes in congenital heart disease. <em>Nature</em>, 498:
220-223.</p>

<p>S. C. Jin, J. Homsy, S. Zaidi, Q. Lu, S. Morton, S. R. DePalma, X. Zeng,
H. Qi, W. Chang, W.-C. Hung, M. C. Sierant, S. Haider, J. Zhang, J.
Knight, R. D. Bjornson, C. Castaldi, I. R. Tikhonoa, K. Bilguvar, S. M.
Mane, S. J. Sanders, S. Mital, M. Russell, W. Gaynor, J. Deanfield, A.
Giardini, G. A. Porter Jr., D. Srivastava, C. W. Lo, Y. Shen, W. S.
Watkins, M. Yandell, H. J. Yost, M. Tristani-Firouzi, J. W. Newburger,
A. E. Roberts, R. Kim, H. Zhao, J. R. Kaltman, E. Goldmuntz, W. K.
Chung, J. G. Seidman, B. D. Gelb, C. E. Seidman2, R. P. Lifton, M.
Brueckner (2017)</p>

<p>Contribution of rare inherited and de novo variants among 2,871
congenital heart disease probands. <em>Nature Genetics</em>, 49: 1593-1601.</p>

<p><strong>[4] Cancer genomics</strong></p>

<p>Y. Zhang, Z. Ouyang, H. Zhao (2017) A statistical framework for data
integration through graphical models with application to cancer
genomics. <em>Annals of Applied Statistics,</em> 11: 161-184.</p>

<p>L. Zeng, J. L. Warren, H. Zhao (2019) Phylogeny-based tumor subclone
identification using a Bayesian feature allocation model. <em>Annals of
Applied Statistics</em>, 13: 1212–1241.</p>

<p>D. Tang, S. Park, H. Zhao (2019) NITUMID: Nonnegative Matrix
Factorization-based Immune-TUmor MIcroenvironment Deconvolution.
<em>Bioinformatics</em>, 36: 1344-1350.</p>

<p>S. Park, H. Xu, H. Zhao (2020) Integrating multidimensional data for
clustering analysis with applications to cancer patient data. <em>Journal
of American Statistical Association</em>, in press.</p>

<p><strong>[5] Neuroscience</strong></p>

<p>Z. Lin, S. Sanders, M. Li, N. Sestan, M. State, H. Zhao (2015) A Markov
random field-based approach to characterizing human brain development
using spatial-temporal transcriptome data. <em>Annals of Applied
Statistics</em>, 9: 429–451.</p>

<p>Y. Zhu, A. M. M. Sousa, T. Gao, M. Skarica, M. Li, G. Santpere, P.
Esteller-Cucala, D. Juan, L. Ferrández-Peral, F. O. Gulden, M. Yang, D.
J. Miller, T. Marques-Bonet, Y. Imamura Kawasawa, H. Zhao, N. Sestan
(2018) Spatiotemporal transcriptomic divergence across human and macaque
brain development. <em>Science</em> 362(6420).</p>

<p><strong>[6] Single cell analysis</strong></p>

<p>S. Park, H. Zhao (2018) Spectral clustering based on learning similarity
matrix. <em>Bioinformatics</em>, 34: 2069-2076.</p>

<p>Y. Liu, J. Warren, H. Zhao (2019) A hierarchical Bayesian model for
single-cell clustering using RNA-sequencing data. <em>Annals of Applied
Statistics</em>, Volume 13: 1733-1752.</p>

<p><strong>[7] Microbiome analysis</strong></p>

<p>T. Wang, H. Zhao (2017) Structured subcomposition selection in
regression and its application to microbiome data analysis. <em>Annals of Applied Statistics,</em> 11: 771-791.</p>

<p>T. Wang, H. Zhao (2017) A Dirichlet-tree multinomial regression model
for associating dietary nutrients with gut microorganisms. <em>Biometrics</em>,
73: 792-801.</p>

<p>T. Wang, H. Zhao (2017) Constructing predictive microbial signatures at
multiple taxonomic levels. <em>Journal of the American Statistical
Association</em>, 112: 1022-1031.</p>

<p>T. Wang, C. Yang, H. Zhao (2019) Prediction analysis for microbiome
sequencing data. <em>Biometrics</em>, 75: 875-884.</p>

<p><strong>[8] Network models</strong></p>

<p>B. Li, H. Chun, H. Zhao (2012) Sparse estimation of conditional
graphical models with application to gene networks. <em>Journal of American
Statistical Association</em>, 107: 152-167.</p>

<p>B. Li, H. Chun, H. Zhao (2014) On an additive semi-graphoid model for
statistical networks with application to pathway analysis. <em>Journal of
American Statistical Association</em>, 109: 1188-1204.</p>

<p>H. Chun, X. Zhang, H. Zhao (2015) Gene regulation network inference with
joint sparse Gaussian graphical models. <em>Journal of Computational and
Graphical Statistics</em>, 24: 954–974<em>.</em></p>

<p>Y. Hu, H. Zhao (2016) CCor: a whole genome network-based similarity
measure between two genes. <em>Biometrics</em>, 72: 1216-1225.</p>

<p>Z. Lin, T. Wang, C. Yang, H. Zhao (2017) On joint estimation of Gaussian
graphical models for spatial and temporal data. <em>Biometrics</em>, 73:
769-779.</p>

<p>K. Lee, B. Li, H. Zhao (2016) On an additive partial correlation
operator and nonparametric estimation of graphical models. <em>Biometrika</em>,
103: 513-530.</p>

<p>K-Y Lee, T. Liu, B. Li, H. Zhao (2020) Learning causal networks via
additive faithfulness. <em>Journal of Machine Learning Research</em>, in press.</p>

      
        <hr />
        <div class="social-share">
  <!--<h4>Share on</h4>
  <ul>
    <li>
      <a href="https://twitter.com/intent/tweet?text=/research/" class="twitter" title="Share on Twitter"><i class="fa fa-twitter"></i><span> Twitter</span></a>
    </li>
    <li>
      <a href="https://www.facebook.com/sharer/sharer.php?u=/research/" class="facebook" title="Share on Facebook"><i class="fa fa-facebook"></i><span> Facebook</span></a>
    </li>
    <li>
      <a href="https://plus.google.com/share?url=/research/" class="google-plus" title="Share on Google Plus"><i class="fa fa-google-plus"></i><span> Google+</span></a>
    </li>
  </ul>-->
</div><!-- /.social-share -->

      
    </div><!-- /.article-wrap -->
    
  </article>
</div><!-- /#index -->

<div class="footer-wrap">
  <footer>
    

<span>&copy; 2020 Hongyu Zhao, Ph.D.. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> using the <a href="https://ncsu-libraries.github.io/jekyll-academic/" rel="nofollow">Jekyll Academic</a> theme.
<br /></span>


  </footer>
</div><!-- /.footer-wrap -->

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="/assets/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
<script src="/assets/js/scripts.min.js"></script>
<script src="/assets/js/openblock.js"></script>




</body>
</html>
